src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +76,17 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    public List<ReadOnlyPerson> sortedListView() {
+        this.internalList.sort(new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
```

Comment:
The naming of o1 and o2 does not follow the java coding standards for naming of variables. It should be in English so that everyone can understand what it is about. Maybe naming it as person1 and person2 would be better.